model/src/main/java/mops/rollen/Dozent.java

```diff
@@ -15,9 +15,14 @@
 @EqualsAndHashCode(of = "uuid")
```

Comment:
I don't think using UUID's is going to work with the now fixed Keycloak. We should use the username instead like we talked about in the Meeting.

Reply:
I'm going to use the Username as ID. Tomorrow

Reply:
Ok